src/figuresClasses.ts

```diff
@@ -1,19 +1,95 @@
-export interface Figure {
+type Shape = 'triangle' | 'rectangle' | 'circle';
+type Color = 'red' | 'green' | 'blue';
 
+export interface Figure {
+  shape: Shape,
+  color: Color,
+  getArea(): number,
 }
 
-export class Triangle {
+export class Triangle implements Figure {
+  shape: Shape = 'triangle';
+
+  constructor(
+    public color: Color,
+    public a: number,
+    public b: number,
+    public c: number,
```

Comment:
It is better not to make fields public when unnecessary, read more on the 'encapsulation OOP principle'